Sources/StreamVideo/Utils/Logger/Logger.swift

```diff
@@ -17,7 +17,7 @@ public struct LogSubsystem: OptionSet {
     }
     
     /// All subsystems within the SDK.
-    public static let all: LogSubsystem = [.database, .httpRequests, .webSocket, .webRTC, .other, .offlineSupport]
+    public static let all: LogSubsystem = [.database, .httpRequests, .webSocket, .webRTC, .other, .offlineSupport, .webRTC]
```

Comment:
`.webRTC` is already there, why do you add it again?

Reply:
Haven't seen it :P